Classes/Issues/EditTitle/EditIssueTitleViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  EditIssueTitleViewController.swift
+//  Freetime
+//
+//  Created by B_Litwin on 10/22/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import GitHubAPI
+import SnapKit
+
+protocol EditIssueTitleViewControllerDelegate: class {
+    func sendEditTitleRequest(newTitle: String, viewController: EditIssueTitleViewController)
+    var currentIssueTitle: String? { get }
+    var viewerCanUpdate: Bool { get }
```

Comment:
See my notes about `IssueManagingContextController` already having access to the model. IMO these last 2 properties can be removed.